content/beginner/090_rbac/create_iam_user.md

```diff
@@ -14,7 +14,7 @@ From within the Cloud9 terminal, create a new user called rbac-user, and generat
 ```
 aws iam create-user --user-name rbac-user
 mkdir tmp
-aws iam create-access-key --user-name rbac-user | tee tmp/create_output.json
+aws iam create-access-key --user-name rbac-user | tee /tmp/create_output.json
```

Comment:
Thank you for the contribution, however, adding "/" won't be correct in all scenarios because the command on L16 which creates the `tmp` directory does not assume the uses current working directory is root.

Reply:
Agreed, either ways works but I don't believe there is value of creating a folder - so I've removed the unneeded mkdir rather than update the following step, thanks for your input!

Reply:
Thank you, looks good to me!